docs/cosa/run.md

```diff
@@ -285,3 +285,35 @@ And point to it and the `core.0` binary:
 ```
 $ cosa run -c --netboot-dir tmp/grub-netboot --netboot boot/grub2/i386-pc/core.0 -m 4096
 ```
+
+### GRUB (Secure Boot)
+
+1. Download a kernel, initramfs, rootfs, shim and GRUB binary
+3. Create an empty disk image (bug in kola): `touch disk.img`
+4. Setup `grub.cfg`
+5. Netboot the result. Be patient as TFTP is slow.
```

Comment:
```suggestion
5. Netboot the result. Be patient as TFTP is slow. Alternatively, use `coreos.live.rootfs_url` to fetch the rootfs over HTTP (see previous example) and remove it from the `initrd` line below.
```

I guess I should've mentioned that `kola http-server` is a thing in these instructions.

Reply:
Ah, I didn't know that existed :). Will take a look.

Reply:
Thanks @jlebon for the pointer, 

- run `cosa shell`, then run `kola http-server` to start http server.
- run `podman exec -it cosa bash`, then run command using `coreos.live.rootfs_url=http://10.0.2.2:8000/live-rootfs.x86_64.img` works.